lib/jxl/render_pipeline/stage_write.cc

```diff
@@ -392,10 +325,15 @@ class WriteToPixelCallbackStage : public RenderPipelineStage {
     run_opaque_ =
         pixel_callback_.Init(num_threads, /*num_pixels=*/kMaxPixelsPerCall);
     JXL_RETURN_IF_ERROR(run_opaque_ != nullptr);
-    temp_.resize(num_threads);
-    for (CacheAlignedUniquePtr& temp : temp_) {
+    tempf_.resize(num_threads);
```

Comment:
IIRC it should be fine if you only create the array for the float case and use it for uint16_t too.

You could also create it for whichever case is actually going to trigger...

Reply:
For unsigned and float16 I currently use both, but for floats there is no need, so I fixed that.